xayn-ai/src/coi/point.rs

```diff
@@ -50,6 +53,26 @@ impl Default for CoiStats {
     }
 }
 
+#[derive(Clone, Deserialize, PartialEq, Serialize)]
+#[cfg_attr(test, derive(Debug))]
```

Comment:
we have to keep in mind that this can increase the binary size, because we have to move some dev-dependencies to dependencies (e.g. like `derivative`).